app/assets/javascripts/widgets/brochure_masthead.js

```diff
@@ -0,0 +1,6 @@
+$(document).ready(function() {
+  $(".offcanvas-menu-trigger a").on("click", function(){
+    $("#js-cu-off-canvas-nav-trigger").trigger("click");
+    return false;
```

Comment:
Also maybe it would make more sense for the js file to be named`brochure.js` as there will probably be more js needed for the rest of the page layout.